server/Arcadia.Assistant.Calendar/SickLeave/SendEmailSickLeaveActor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Arcadia.Assistant.Calendar.SickLeave.Events;
+using System;
+using Akka.Actor;
+using Akka.DI.Core;
+using MimeKit;
+
+namespace Arcadia.Assistant.Calendar.SickLeave
+{
+    using MailKit.Net.Smtp;
+    using MailKit.Security;
+    using Microsoft.Extensions.Configuration;
+
+    public class SendEmailSickLeaveActor : UntypedActor
+    {
+        private readonly IConfigurationSection mailConfig;
+        public SendEmailSickLeaveActor(IConfigurationSection mailConfig)
+        {
+            this.mailConfig = mailConfig;
+        }
+
+        public static Props GetProps() => Context.DI().Props<SendEmailSickLeaveActor>();
+
+        protected override void OnReceive(object message)
+        {
+            switch (message)
+            {
+                case SickLeaveIsApproved ev:
+                    SendEmail(ev);
+                    break;
+            }
+        }
+
+        private void SendEmail(SickLeaveIsApproved Event)
+        {
+            var user = mailConfig["SmtpUser"];
+            var pass = mailConfig["SmtpPass"];
+
+            using (var client = new SmtpClient())
+            {
+                var body = "Dear Sir/Madam,\nNotify you, that employee " + Event.UserId +
```

Comment:
1) move both body and subject into a configuration file.
2) in all languages, try to avoid string concatenation if possible. Use string.Format or, better, string interpolation

Reply:
If you move it in template, string.Format will be used because the configuration will look as 'aaaaaa from {0} says {1}'